### ISSUE TITLE
Cannot scroll down the preview tab

### ISSUE THREAD

**Issue by tientaidev** [mr-pennyworth/alfred-better-dictionaries#14]
![image](https://user-images.githubusercontent.com/43933775/150673221-042f421d-56f9-4d0c-b803-0bd1b0472f7d.png)

I cannot scroll down the preview tab. When the mouse hover over the preview tab, it automatically jumps the random definition.

**Comment by yarub123** [comment]
@tientaidev Yeah this seems to be a scaling issue. I run into this very thing. You have to tinker with the theme. And when you tinker with the theme you have to play around and try diff settings for whatever resolution you are using on your monitor. 

Moving the Alfred launcher towards the bottom/middle of the screen can help fix a lot of these jumps you are seeing.

Also if you have an external monitor, you will have issues when switching to it (like if you're on a laptop and you plug in the external). Idk why, but the shift in different resolutions seems to mess it up. 

In that scenario you would have to kill alfred. If that doesn't solve it, kill alfred pane and alfred and start them back up. 

If that doesn't fix it, you can try adjusting the theme under "Appearances" Alfred settings and you should be able to figure out which spot fixes it. 

It's annoying af, I'll admit and I still haven't pinpointed the exact cause. 

Maybe you don't even have this issue anymore?
